canonical-web-and-design/maas-ui
# Add WebSocket connection

Done:
- Added a websocket client and added sagas to send/receive messages. Fixes: https://github.com/canonical-web-and-design/maas-squad/issues/1313
- Added a basic repository list using the connection.

QA:
- Modify your MAAS to not require CSRF tokens for the websocket API (see me at my desk).
- Log into MAAS.
- Open index.html in maas-ui and modify "websocket_url" to point to a MAAS.
- ./run maas-ui
- Visit /repositories
- You should see a list of repositories.
